### PR TITLE
feat: List process registry command

### PR DESCRIPTION
**Pull request by alexrodfe**
## Description

- Added list process registry command

## Motivation and Context

Users will be able to list process registries from a desired product. This list can also be filtered by process type.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [X] I have created tests for my code changes, and the tests are passing.
- [X] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by kafkaphoenix — DISMISSED** [review]

File: api/process-registry/list.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import (
+	"github.com/konstellation-io/kli/internal/services/configuration"
+)
+
+func (c *processRegistryClient) List(
+	server *configuration.Server, productID, processType string,
+) ([]ProcessRegistry, error) {
+	query := `
+		query ProcessRegistries($productID: ID!, $processType: String) {
+			processRegistries(productID: $productID, processType: $processType) {
```
Comment (dfinteligenz) [inline api/process-registry/list.go:12]:
```suggestion
			registeredProcesses(productID: $productID, processType: $processType) {
```

File: cmd/process-registry/process_registry.go [hunk 2]
```diff
@@ -13,13 +13,14 @@ func NewProcessRegistryCmd(logger logging.Interface) *cobra.Command {
 		Use:   "process-registry <command>",
 		Short: "Manage process-registry",
 		Example: heredoc.Doc(`
-			$ kli process-registry ls [opts...]
+			$ kli process-registry ls <product_id> [--type <process_type>] [opts...]
```
Comment (dfinteligenz) [inline cmd/process-registry/process_registry.go:16]:
```suggestion
			$ kli process-registry ls <product_id> [opts...]
```

File: internal/render/interface.go [hunk 3]
```diff
@@ -14,4 +15,5 @@ type Renderer interface {
 	RenderWorkflows(workflows []krt.Workflow)
 	RenderProcesses(processes []krt.Process)
 	RenderConfiguration(scope string, config map[string]string)
+	RenderProcessRegistries(registries []registry.ProcessRegistry)
```
Comment (dfinteligenz) [inline internal/render/interface.go:18]:
```suggestion
	RenderRegisteredProcesses(registries []registry.ProcessRegistry)
```

**Review by dfinteligenz — CHANGES_REQUESTED** [review]

**Review by JYisus — APPROVED** [review]

**Comment by alexrodfe** [comment]
Moved to https://github.com/konstellation-io/kai-kli/pull/35

**Pull request closed without merging**